packages/react-ui/src/Tile/index.js

```diff
@@ -8,7 +8,8 @@ const StyledTile = styled.a`
   position: relative;
   display: flex;
   flex-direction: column;
-  align-items: flex-start;
+  flex-wrap: wrap;
+  align-items: start;
```

Comment:
```suggestion
  align-items: flex-start;
```

Reply:
https://developer.mozilla.org/fr/docs/Web/CSS/align-items `start` is ok ?